Move.toml

```diff
@@ -0,0 +1,6 @@
+[package]
+name = "MoveStdlib"
```

Comment:
This is how we use it nowadays:
```
MoveStdlib = { git = "https://github.com/eigerco/substrate-move.git", subdir = "language/move-stdlib", rev = "main" }
substrate-stdlib = { git = "https://github.com/eigerco/substrate-stdlib.git", rev = "main" }
```

How do you feel if we rename `MoveStdlib` to `move-stdlib` - or the opposite,
we rename `substrate-stdlib` to `SubstrateStdlib`.
It would be nice to have alignment here

Reply:
Let's stick with MoveStdlib - seems to be used everywhere.

> we rename substrate-stdlib to SubstrateStdlib.

We can do this instead then.